### ISSUE TITLE
HeskPlugin_LDAP Breaks the Hesk 3.5.0

### ISSUE THREAD

**Issue by the-aLeXiS** [Kinderschutzbund-Karlsruhe/HeskPlugin_LDAP#2]
After installation and setup the hesk/admin doesn't open, without any errors, just blank screen.
Hesk's logs are silent about this

**Comment by nini57** [comment]
Hello, have you solved this issue ?

**Comment by the-aLeXiS** [comment]
> Hello, have you solved this issue ?

Still unluck

**Comment by pcadminby** [comment]
the problem is that the script patches the **file admin_functions.inc.php** and apparently does it incorrectly.. to restore functionality, you can take the original file. However, the question remains open how to install this necessary plugin in
3.5.2

**Comment by reko77** [comment]
Yes, I also encountered the same problem. After deploying the plugin, my admin section was empty without errors. When I overwrote admin_functions.inc.php. But the necessary actions such as AD synchronization did not take place. It would be good if the author commented on this and possibly made a correction.

**Comment by domidodo** [comment]
The problem was that Hesk moved the login logic from one file to another. As a result, the wrong file was modified, which caused errors.

The issue has now been resolved.
I also added a check to ensure that this won’t happen again in the future.

You can either reinstall Hesk, or simply overwrite the "/admin/index.php" and "/inc/admin_functions.inc.php" files with the original versions from the Hesk zip file.
After that, you can reinstall the plugin.
